packages/react/src/components/form/Switch/Switch.tsx

```diff
@@ -1,97 +1,65 @@
-import { PadlockLockedFillIcon } from '@navikt/aksel-icons';
-import cl from 'clsx/lite';
 import type { InputHTMLAttributes, ReactNode } from 'react';
 import { forwardRef } from 'react';
 
-import { omit } from '../../../utilities';
 import { Label } from '../../Label';
-import { Paragraph } from '../../Paragraph';
-import type { FormFieldProps } from '../useFormField';
-
-import { useSwitch } from './useSwitch';
+import { ValidationMessage } from '../../ValidationMessage';
+import { Field, type FieldProps } from '../Field';
+import { Input, type InputProps } from '../Input';
 
 export type SwitchProps = {
-  /** Switch label */
-  children?: ReactNode;
+  /** Optional aria-label */
+  'aria-label'?: string;
+  /** Radio label */
+  label?: ReactNode;
+  /** Description for field */
+  description?: ReactNode;
   /** Value of the `input` element */
-  value?: string;
-  /** Position of switch around the label
-   * @default left
+  value?: InputProps['value'];
+  /** Validation message for field */
+  validation?: ReactNode;
```

Comment:
This is a new feature and Users have no way of toggling which validation color/severity this is. Suggest removing and add it later if needed?